code/headers/mlx90640_processor.hpp

```diff
@@ -0,0 +1,125 @@
+#pragma once
+
+#include <array>
+#include <cmath>
+#include <mlx90640_i2c.hpp>
+
+namespace r2d2::thermal_camera {
+    /**
+     * MLX90640 16 bits registers
+     * INTERNAL means internal register.
+     * EE means EEPROM (Electric Erasable Programmable Read-Only)
+     * RAM means just RAM (Random Acces Memory)
+     **/
+    enum : uint16_t {
```

Comment:
Could you move this out of namespace scope? Currently these values will bleed into scope upon including this file.

Reply:
FTFY